src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        return String.valueOf(Colors.values()[new Random().nextInt(Colors.values().length)]);
```

Comment:
Use the 'name()' method to get the String representation of enum constants instead of 'String.valueOf()'. The 'name()' method is the preferred way to get the name of an enum constant as it will always return the exact name as declared in the enum type.